src/popup.html

```diff
@@ -60,6 +60,8 @@
             </button>
           </div>
         </div>
+        <div class="form-text text-center" id="errorText" style="display: none">
```

Comment:
Consider using Bootstrap for coloring
`<p class="text-danger">.text-danger</p>`